test/functional/wiki_controller_test.rb

```diff
@@ -108,21 +108,21 @@ def teardown
 
   test "viewing edit wiki page" do
 
-    get :edit, 
+    get :edit,
          id: 'organizers'
```

Comment:
Also look how the id for the edit action should be the page slug, not the numeric id! That should solve your last issue.